NEWS.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.15.23]
+
+### Added
+
+* a field `point` to `ValidationFibreVector` to store potentially store the point of the vector.
+* a field `store_base_point` to `ValidationManifold` to indicate whether for new fibre vectors the base point should be stored.
+
+### Changed
+
+* the internal function `_array_value` was renamed to `_value`,
```

Comment:
Manifolds.jl currently uses and extends `array_value`. It would be easier to just keep the name, even if it isn't entirely appropriate.

Reply:
Since we have an `_` I would consider that internal and since it also works for arbitrary points and not only for array values (now even more precise), I think it is worth renaming.

That it breaks Manifolds.jl is a bit of a problem, but we were maybe not careful enough splitting both packages there.
I can do the PR that fixes that on Manifolds, if you prefer.

Reply:
Actually, the old name is `array_value` (without the leading underscore). And, as you can see, making it internal doesn't exactly work out since there is no way to extend `ValidationManifold` without using that function.

Renaming to `_value` IMO overly emphasizes the level of "being internal" this function has. There is not much reason to extend it and its direct usage is rarely the right choice, but it's not "do not touch, copy if you must" level of "internal".

Reply:
Hm, right to extend that to other functions is indeed complicated from outside (new types/manifolds should work directly). But the name is super misleading, it does not provide the array value but the internally stored thingy – however another user typed that, maybe an array, maybe an `AbstractManifoldPoint´ or – whatever..

can we then maybe (for non-breakingness) keep but deprecate `array_value` and use a new name?
`internal_value` maybe?
The next breaking release of ManifoldsBase (or the second-to-next) could then remove it

Reply:
OK, `internal_value` sounds fine, and `array_value` can be a deprecated alias.